docs/DeveloperGuide.md

```diff
@@ -152,6 +152,7 @@ is not specified, an `InvalidCommandException` will be thrown.
 ### Overview
 * [Getting User Input Continuously](#getting-user-input-continuously)
 * [Parsing User Input and Getting the Right Command](#parsing-user-input-and-getting-the-right-command)
+    * [Illegal Characters & Phrases](#illegal-characters--phrases)
```

Comment:
Is this link not working?

Reply:
i think there is an extra "-" between characters and phrases

Reply:
> Is this link not working? - @a1021492980 

Thanks for pointing it out. Funnily it worked on my local machine, but when I just tested it didn't work... weird.

Anyway, I've fixed the broken links in Commit 5aaaeeb by replacing the ampersand with the word 'and'. Please try it out and see if it works now. 🙂 


> i think there is an extra "-" between characters and phrases - @emilysim00 

I think the extra dash is due to the ampersand symbol not being usable for reference links. I've changed it to the word 'and' instead.